pokete_classes/movemap.py

```diff
@@ -25,8 +26,8 @@ def __init__(self, height, width):
         self.balls_label = se.Text("")
         self.label_bg = se.Square(" ", self.width, 1, state="float")
         self.label = se.Text(
-                        "1: Deck  2: Exit  3: Map  4: Inv.  5: Dex  6: Clock  "
-                        "?: Help")
+            "1: dEck  2: Quit  3: Map  4: Inv.  5: Pokedex  6: Clock  7: menU  ?: help"
```

Comment:
```suggestion
            "1: dEck  2: Quit  3: Map  4: Inv.  5: Pokedex  6: Clock  7: menU  ?: help"
```
This does not work out with minimum window size.

Reply:
So it won't work with your change either, right? You didn't actually change it. Menu needs to be removed again I guess.